app/src/sharedTest/java/org/oppia/app/profile/AdminAuthActivityTest.kt

```diff
@@ -132,6 +135,87 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_buttonState_ConfigurationChanged_ButtonStateIsPreserved() {
+    launch<AdminAuthActivity>(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context,
+        "12345",
+        -10710042
+      )
+    ).use {
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
+        typeText("12345"),
+        closeSoftKeyboard()
+      )
+      it.onActivity {
+        it.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        assert(it.requestedOrientation == ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+      }
+      onView(withId(R.id.submit_button)).check { view, noViewFoundException ->
```

Comment:
We actually do not check the drawable in this format. Please check other test cases in code on how to match drawables. You can use this function too: tests package -> utitlity -> EspressoTestMatchers -> withDrawable()

Reply:
Done